src/any_llm/providers/gemini/interactions.py

```
@@ -0,0 +1,559 @@
+        input_tokens=input_tokens,
+        input_tokens_details=InputTokensDetails(
+            cached_tokens=int(cached_tokens or 0),
+            cache_write_tokens=0,
+        ),
+        output_tokens=output_tokens,
+        output_tokens_details=OutputTokensDetails(reasoning_tokens=int(reasoning_tokens or 0)),
+        total_tokens=total_tokens,
+    )
+
+
+def _map_status(status: Any) -> Literal["completed", "failed", "in_progress", "cancelled", "queued", "incomplete"]:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `status`

(ANN401)